packages/clerk-js/src/ui/components/UserProfile/Web3Form.tsx

```diff
@@ -62,7 +62,7 @@ export const AddWeb3WalletActionMenu = withCardStateProvider(() => {
           onClick={() => connect(strategy)}
           isLoading={card.loadingMetadata === strategy}
           isDisabled={card.isLoading}
-          localizationKey={`Connect ${strategyToDisplayData[strategy].name} wallet`}
+          localizationKey={`Connect ${strategyToDisplayData[strategy].name}`}
```

Comment:
I believe it's time to create a proper localization key here

Reply:
I went ahead and aligned this with the Social connections section. What's changed is:
1. Update the copyright of the primary button from `Web3 wallets` to `Connect wallet`
2. Introduced a new localization key `userProfile.web3WalletPage.web3WalletButtonsBlockButton` to only render the web3 provider name on the dropdown